tests/Cache/FilesystemTest.php

```diff
@@ -81,7 +81,7 @@ public function testWrite()
 
     public function testWriteFailMkdir()
     {
-        $this->expectException('\RuntimeException');
+        $this->expectException(\RuntimeException::class);
```

Comment:
Good catch, I had to stringify classes because of PHP 5.4. 
But then we removed it and I forgot to revert it